### ISSUE TITLE
Advanced settings pane is not accessible with screen-readers

### ISSUE THREAD

**Issue by kylehakala** [devshawn/password.live#32]
Advanced page Password Settings panel is lacking in compatibility with screen-readers

Tested on macOS Using (Browser + Screenreader): 
- Chrome + ChromeVox
- Chrome + macOS VoiceOver
- Safari + macOS VoiceOver
- iOS Safari + iOS VoiceOver
- FireFox + macOS VoiceOver

Examples of some issues: 
- ChromeVox for Chrome does not provide an audible label on any Password Settings but they can be changed after hearing some audible feedback on whether or not a checkbox is checked. Regardless of the setting in focus, no label is read aloud by the screen reader for any setting. 
- iOS VoiceOver on Safari speaks the current value (either the checkbox state, or the length number that is displayed, but not via selecting the slider). iOS Voiceover also speaks aloud the visual dividers between each setting. 
-  macOS VoiceOver in Safari can alter the checkboxes and speak the labels of all checkboxes, but does not speak the label for the Length setting in any capacity when tabbing through the page. macOS VoiceOver will speak certain page elements if they are explicitly clicked on in Safari. 
- macOS VoiceOver in Chrome can alter the checkboxes and length slider, but no labels are spoken, even if explicitly clicked on. 
- Firefox with macOS VoiceOver did not allow tabbing through even the first page element via the VoiceOver selector. Page elements were highlighted visually when tabbing, but the VoiceOver selector did not enter the page, so nothing was spoken aloud from the page content.
